docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/kie-server-was-environment-set-proc.adoc

```diff
@@ -52,17 +52,33 @@ ifdef::DM[]
 endif::DM[]
 
 ifdef::BA[]
+|`kie.services.jms.queues.response`
+|`jms/KIE.RESPONSE.ALL`
+|The JNDI name of JMS queue for responses used by the {CENTRAL} and {KIE_SERVER}.
+
+|`org.uberfire.domain`
+|`WSLogin`
+|JAAS `LoginContext` domain used to authenticate users when using JMS.
+
 |`org.kie.server.persistence.ds`
-|Set according to database type and configuration.
-|Datasource JNDI name.
+|`jdbc/jbpm`
+|Data source JNDI name for {KIE_SERVER}.
 
 |`org.kie.server.persistence.tm`
 |`org.hibernate.service.jta.platform.internal.WebSphereExtendedJtaPlatform`
 |Transaction manager platform for setting Hibernate properties.
 
 |`org.kie.server.persistence.dialect`
-|Set according to database type and configuration.
-|Specifies the Hibernate dialect to be used.
+|Example: `org.hibernate.dialect.H2Dialect`
+|Specifies the Hibernate dialect to be used. Set according to data source.
+
+|`org.kie.executor.jms.queue`
+|`jms/KIE.EXECUTOR`
+|Use the same job executor JMS queue for {CENTRAL} and {KIE_SERVER}.
```

Comment:
I'd remove {CENTRAL} here as it's no longer used - no execution server in business central